pkg/blobstore/local/file_backed_location_record_array.go

```diff
@@ -0,0 +1,88 @@
+package local
+
+import (
+	"encoding/binary"
+	"sync"
+
+	"github.com/buildbarn/bb-storage/pkg/filesystem"
+)
+
```

Comment:
Maybe it's good to add a `const` for the number 60?
```go
const (
   FileBackedLocationRecordSize = len(LocationRecord.Key.Digest) + 4 + 8 + 8 + 8
)
```
It's good to make this public, because pkg/blobstore/configuration also depends on it.

Reply:
For some reason using `LocationRecord.Key.Digest` gave errors, so I used the Sha256.Size directly.